plugin/chatgpt.vim

```diff
@@ -49,6 +49,20 @@ if !exists("g:chat_gpt_split_direction")
   let g:chat_gpt_split_direction = 'horizontal'
 endif
 
+let g:prompt_templates = {
+\ 'rewrite': 'Can you rewrite it more idiomatically?',
+\ 'review': 'Can you provide a code review for?',
+\ 'document': 'Return documentation following language pattern conventions.',
+\ 'explain': 'Can you explain it?',
+\ 'test': 'Can you write a test for it?',
+\ 'fix': 'It has an error I need you to fix.'
+\}
+
+if exists('g:chat_gpt_custom_prompts')
+  call extend(g:prompt_templates, g:chat_gpt_custom_prompts)
+endif
```

Comment:
I was thinking it might be good to move these closer to where the commands are defined so it is easier to follow the logic

Reply:
Stylistically I prefer things that are global/constants to live at the top.

Reply:
Yeah this is also a good point. You could think about splitting the file to multiples where you have the function definitions and other general code in the first file and the default definitions in the second one. On the other hand this might make the readability even harder when the globals are defined in another file.